src/dotnet/APIView/APIViewWeb/Controllers/CommentsController.cs

```diff
@@ -22,14 +22,15 @@ public CommentsController(CommentsManager commentsManager, ReviewManager reviewM
         }
 
         [HttpPost]
-        public async Task<ActionResult> Add(string reviewId, string revisionId, string elementId, string commentText)
+        public async Task<ActionResult> Add(string reviewId, string revisionId, string elementId, string commentText, bool sampleComment = false)
         {
             var comment = new CommentModel();
             comment.TimeStamp = DateTime.UtcNow;
             comment.ReviewId = reviewId;
             comment.RevisionId = revisionId;
             comment.ElementId = elementId;
             comment.Comment = commentText;
+            comment.IsSampleComment = sampleComment;
```

Comment:
Praveen might think otherwise, but I wonder if this should be `usageSampleComment` and `isUsageSampleComment`, just to make it clear (otherwise people might wonder what a 'sample comment' is).